ui/tests/acceptance/enterprise-replication-test.js

```diff
@@ -59,6 +59,8 @@ module('Acceptance | Enterprise | replication', function (hooks) {
     await pollCluster(this.owner);
     await settled();
     await click('[data-test-replication-link="secondaries"]');
+    await click(GENERAL.button('Copy token'));
```

Comment:
The release notes show some patch changes to the `Modal` component that it looks like affected these tests. Now the modal is explicitly closed (copying the token has to happen to do so) before moving on to the next test action.